extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -155,6 +159,9 @@ export class AmpSidebar extends AMP.BaseElement {
       element.setAttribute('side', this.side_);
     }
 
+    const width = element.getAttribute('width');
+    setImportantStyles(element, {'--sidebar-width': `${width}px`});
```

Comment:
Is it possible for the sidebar width to change after the first layout? If so we should probably update this inside a window `resize` listener

Reply:
Sidebar currently has min-width (45px) and max-width (80vw) properties so its actual width might change, but I think the width property should stay constant on window resize.